slxos/playbooks/linkagg/site.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: network
+  connection: network_cli
```

Comment:
Since the hosts have this specified in the host_vars, don't need `network_cli` here.